analytics/terraform/emr-eks-karpenter/addons.tf

```diff
@@ -15,20 +20,8 @@ module "eks_blueprints_kubernetes_addons" {
   enable_amazon_eks_aws_ebs_csi_driver = true
 
   #---------------------------------------
-  # Metrics Server
+  # Kubernetes Add-ons
   #---------------------------------------
-  enable_metrics_server = true
```

Comment:
Should we keep metrics server? Prometheus cannot collect kube-state-metrics without metrics server.

Reply:
Good point. Should we build kube-state-metrics instead? Fee customers are using this with Prometheus Adaptor